### PR TITLE
Add Llama support to token tracker and more openai model prices

### PR DESCRIPTION
**Pull request by kongzii**

### CODE REVIEW

File: microchain/models/openai_generators.py [hunk 1]
```diff
@@ -1,26 +1,5 @@
 from termcolor import colored
-
-class TokenTracker:
```
Comment (kongzii) [inline microchain/models/openai_generators.py:3]: Moved to its own file, as it's not OpenAI-exclusive.

File: microchain/models/token_tracker.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        self.prompt_tokens += usage.prompt_tokens
+        self.completion_tokens += usage.completion_tokens
+
+    def get_total_cost(self, model):
+        # From https://openai.com/pricing
+        if model in ["gpt-4-0125-preview", "gpt-4-1106-preview", "gpt-4-turbo-preview"]:
+            costs = {"prompt": 1e-05, "completion": 3e-05}
+        elif model == "gpt-3.5-turbo-0125":
+            costs = {"prompt": 5e-07, "completion": 1.5e-06}
+        elif model == "meta/meta-llama-3.1-405b-instruct":
```
Comment (kongzii) [inline microchain/models/token_tracker.py:23]: Everything is the same as before in this class, just added more models here

**Comment by galatolofederico** [comment]
Amazing thanks! do you need a version bump?

**Comment by kongzii** [comment]
Thanks for merging! Sure, it's always nice to install from pip 🙏

**Pull request merged**